src/redux/books/Book.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+function Book(props) {
+  const { category, title, author } = props;
+  return (
+    <>
+      <div className="Book">
+        {category}
+        <br />
+        {title}
+        <br />
+        {author}
+      </div>
+      <div className="Actions">
+        <button type="button">Remove</button>
+      </div>
+    </>
+  );
+}
+
```

Comment:
Please remove the `book.js` component and `books.css` file in the Redux directory. It unnecessary to have them in the Redux file structure. They belong in the `components` directory.